subworkflows/local/fastq_download_fastq_dl/main.nf

```diff
@@ -0,0 +1,34 @@
+include { FASTQ_DL          } from '../../../modules/local/fastq_dl/main'
+include { PREFETCH_CHECKER  } from '../../../modules/local/prefetchchecker/main'
+
+//
+// Download FASTQ sequencing reads from ENA with fastq-dl
+//
+workflow FASTQ_DOWNLOAD_FASTQ_DL {
+    take:
+    ch_ids   // channel: [ val(meta), val(id) ]
```

Comment:
See my comment earlier about having `meta.insdc_accession = meta.insdc_accession[0]` and `id = meta.insdc_accession[0]`.